src/cbmc_starter_kit/template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -287,11 +291,25 @@ async def configure_proof_dirs(
         profiling = [
             "ENABLE_MEMORY_PROFILING=true"] if enable_memory_profiling else []
 
-        proc = await asyncio.create_subprocess_exec(
-            # Allow interactive tasks to preempt proof configuration
-            "nice", "-n", "15", "make", *pools, *profiling, "-B", "--quiet",
-            "_report", cwd=path)
-        await proc.wait()
+        # Allow interactive tasks to preempt proof configuration
+        if debug:
+            proc = await asyncio.create_subprocess_exec(
+                "nice", "-n", "15", "make", "VERBOSE=1", *pools, *profiling,
+                "-B", "_report", cwd=path,
+                stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE)
+            stdout, stderr = await proc.communicate()
+            logging.debug("stdout:")
+            for line in stdout.decode().splitlines():
+                logging.debug(line)
+            logging.debug("stderr:")
+            for line in stderr.decode().splitlines():
+                logging.debug(line)
+        else:
+            proc = await asyncio.create_subprocess_exec(
+                "nice", "-n", "15", "make", *pools, *profiling, "-B", "--quiet",
+                "_report", cwd=path)
+            await proc.wait()
+
```

Comment:
Don't you want to use a single block of code (not if-then-else) and just use 
```
"VERBOSE=1" if debug else ""
```
The debug output will be generated only when the logging level is set to debug.

Reply:
> The debug output will be generated only when the logging level is set to debug.

That is right. The if-then-else block was indeed unnecessary.